bindings/python/src/expose-qpobject.hpp

```diff
@@ -22,72 +23,62 @@ namespace python {
 
 template<typename T>
 void
-exposeQpObjectDense(pybind11::module_ m)
+exposeQpObjectDense(nanobind::module_ m)
 {
-  ::pybind11::enum_<DenseBackend>(m, "DenseBackend", pybind11::module_local())
+  ::nanobind::enum_<DenseBackend>(m, "DenseBackend")
     .value("Automatic", DenseBackend::Automatic)
     .value("PrimalDualLDLT", DenseBackend::PrimalDualLDLT)
     .value("PrimalLDLT", DenseBackend::PrimalLDLT)
     .export_values();
 
-  ::pybind11::enum_<HessianType>(m, "HessianType", pybind11::module_local())
+  ::nanobind::enum_<HessianType>(m, "HessianType")
     .value("Dense", proxsuite::proxqp::HessianType::Dense)
     .value("Zero", proxsuite::proxqp::HessianType::Zero)
     .value("Diagonal", proxsuite::proxqp::HessianType::Diagonal)
     .export_values();
 
-  // ::pybind11::class_<proxsuite::proxqp::dense::preconditioner::RuizEquilibration<T>>(m,
-  // "ruiz", pybind11::module_local())
-  //   .def(::pybind11::init(), "Default constructor.")
-  //   .def_readwrite("mu_eq", &RuizEquilibration<T>::delta)
-  //   .def_readwrite("mu_in", &RuizEquilibration<T>::c)
-  //   .def_readwrite("rho", &RuizEquilibration<T>::dim)
-  //   .def_readwrite("iter", &RuizEquilibration<T>::epsilon)
-  //   .def_readwrite("iter_ext", &RuizEquilibration<T>::max_iter)
-  //   .def_readwrite("run_time", &RuizEquilibration<T>::sym);
+  // ::nanobind::class_<proxsuite::proxqp::dense::preconditioner::RuizEquilibration<T>>(m,
+  // "ruiz")
+  //   .def(::nanobind::init(), "Default constructor.")
+  //   .def_rw("mu_eq", &RuizEquilibration<T>::delta)
+  //   .def_rw("mu_in", &RuizEquilibration<T>::c)
+  //   .def_rw("rho", &RuizEquilibration<T>::dim)
+  //   .def_rw("iter", &RuizEquilibration<T>::epsilon)
+  //   .def_rw("iter_ext", &RuizEquilibration<T>::max_iter)
+  //   .def_rw("run_time", &RuizEquilibration<T>::sym);
 
-  // ::pybind11::class_<proxsuite::proxqp::dense::preconditioner::RuizEquilibration<T>>(m,
-  // "ruiz", pybind11::module_local())
-  //   .def(::pybind11::init(), "Default constructor.")
-  //   .def_readwrite("mu_eq", &RuizEquilibration<T>::delta)
-  //   .def_readwrite("mu_in", &RuizEquilibration<T>::c)
-  //   .def_readwrite("rho", &RuizEquilibration<T>::dim)
-  //   .def_readwrite("iter", &RuizEquilibration<T>::epsilon)
-  //   .def_readwrite("iter_ext", &RuizEquilibration<T>::max_iter)
-  //   .def_readwrite("run_time", &RuizEquilibration<T>::sym);
+  // ::nanobind::class_<proxsuite::proxqp::dense::preconditioner::RuizEquilibration<T>>(m,
+  // "ruiz")
+  //   .def(::nanobind::init(), "Default constructor.")
+  //   .def_rw("mu_eq", &RuizEquilibration<T>::delta)
+  //   .def_rw("mu_in", &RuizEquilibration<T>::c)
+  //   .def_rw("rho", &RuizEquilibration<T>::dim)
+  //   .def_rw("iter", &RuizEquilibration<T>::epsilon)
+  //   .def_rw("iter_ext", &RuizEquilibration<T>::max_iter)
+  //   .def_rw("run_time", &RuizEquilibration<T>::sym);
```

Comment:
this here seems to be duplicated and commented... maybe we just remove it ? or at least one version

Reply:
I'm not actually sure why this was commented initially